.github/workflows/pull_request.yaml

```diff
@@ -15,4 +15,4 @@ jobs:
       - uses: DeterminateSystems/magic-nix-cache-action@main
         with:
           upstream-cache: https://ngi.cachix.org/
-      - run: nix flake check
+      - run: nix flake check --debug --print-build-logs
```

Comment:
```suggestion
      - run: nix flake check --verbose --print-build-logs
```

Reply:
This was added for testing as we cannot replicate the failure locally; we will probably revert this change when the issue is resolved.